src/tle-update.c

```diff
@@ -396,10 +396,10 @@ void tle_update_from_files(const gchar * dir, const gchar * filter,
             /* store time of update if we have updated something */
             if ((updated > 0) || (newsats > 0))
             {
-                GTimeVal        tval;
+		gint64          now;
```

Comment:
In the future, please keep 4-spaces indentation.

Reply:
ah, do you want all spaces? It looks like it. I'll make sure I'll follow it.  Would you take a commit to fix the whitespace errors?

While we are at it, what do you think about trailing whitespaces.  There are so many.  Do you think we can remove them in one go or gradually removing while fixing the line with it?

> Furthermore, until we get a formal coding style guide, please try to follow the coding
style used in the surrounding code.

What do you think about providing a config file for clang-format or uncrustify?

Is it ok to talk about it here, or create an issue about it? How about enabling github discussion?

Reply:
> While we are at it, what do you think about trailing whitespaces. There are so many. Do you think we can remove them in one go or gradually removing while fixing the line with it?

I prefer that changes that fix the formatting or style of previously written code to be done separately.

> What do you think about providing a config file for clang-format or uncrustify?

There is a config file for the [indent](https://www.gnu.org/software/indent/manual/indent.html) tool, however, I don't remember whether all files have been indented already. I might switch to clang-format or some other tool in the future, but all these kind of things are time-consuming and I have other priorities for the time being.

> Is it ok to talk about it here, or create an issue about it? How about enabling github discussion?

Thanks for the tip, I wasn't aware of github discussions but I will check it out.